src/content/docs/style-guide/word-choice/user-related-language-guidelines.mdx

```diff
@@ -96,7 +96,7 @@ Example use:
 *The ability to do this is governed by the "modify APM settings" [capability](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#capabilities).*   </td>
 
       <td>
-One way to reference a capability is by using quotes as shown in the example here, but we don't have firm rules on how to reference capabilities. One reason for this is that capabilities don't have specific proper names and are more informal. To see what we mean, see the screenshot of the capabilities UI table. The rows in the capabilities table have a heading (like "APM"), and refer to a feature (like "Application settings"), and have columns referencing more granular abilities (like "Create" or "Delete"). For this reason, we don't have a firm rule but the goal is simply to try to make it clear which capability you're referencing. 
+One way to reference a capability is by using quotes as shown in the example here, but we don't have firm rules on how to reference capabilities. One reason for this is that capabilities don't have specific proper names and are more informal. To see what we mean, see the [screenshot of the capabilities UI table](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#capabilities). The rows in the capabilities table have a heading (like "APM"), and refer to a feature (like "Application settings"), and have columns referencing more granular abilities (like "Create" or "Delete"). For this reason, we don't have a firm rule but the goal is simply to try to make it clear which capability you're referencing. 
```

Comment:
Second to last sentence, may want to consider turning into bullets. Or else delete the first "and"
Last sentence add a comma "firm rule, but the goal"